packages/main/src/Menu.ts

```diff
@@ -542,9 +546,16 @@ class Menu extends UI5Element {
 		this._parentItemsStack.push(item);
 	}
 
+	_onfocusin(e: CustomEvent<any>): void {
```

Comment:
Just noticed the `CustomEvent<any>`. Please import the event type from the component that exposes the `_focused` event and use this here.

Reply:
done